packages/perseus/src/widgets/interactive-graphs/graphs/screenreader-text.ts

```diff
@@ -1,7 +1,11 @@
-export function srFormatNumber(a: number, locale: string): string {
+export function srFormatNumber(
+    a: number,
+    locale: string,
+    maximumFractionDigits?: number,
```

Comment:
Adding this parameter so that the screen reader label for the side angles can match what's presented by the "show angles" button (which is only one decimal point).